model_evaluation/score_test_data.py

```diff
@@ -6,7 +22,7 @@
 import tensorflow as tf
 
 import input_fn_example
-from utils_export.dataset import Dataset, Model
+from utils_export.dataset import Dataset, Model, CMLE_QUOTA_PREDICTION
```

Comment:
Why is this imported in this file if it's not used here?

Reply:
Good point, removed quota here